lib/ukmpi/Makefile.uk

```diff
@@ -4,3 +4,4 @@ CINCLUDES-$(CONFIG_LIBUKMPI)   += -I$(LIBUKMPI_BASE)/include
 CXXINCLUDES-$(CONFIG_LIBUKMPI) += -I$(LIBUKMPI_BASE)/include
 
 LIBUKMPI_SRCS-$(CONFIG_LIBUKMPI_MBOX) += $(LIBUKMPI_BASE)/mbox.c
+LIBUKMPI_SRCS-$(CONFIG_LIBUKMPI_MBOX) += $(LIBUKMPI_BASE)/isr.c|isr
```

Comment:
I would suggest to not call the file `isr.c` because it implements the mbox components as isr-safe version. What do you think of `mbox_isr.c`?